### DISCUSSION TITLE
just a question about fuel gauge

### DISCUSSION THREAD

**Discussion by grsthegreat** [jgyates/genmon#1080] [Questions] [answered]
### Steps to Reproduce

i want to get rid of fuel gauge

### Expected Result

i no longer have a tank monitor on propane tank and dont want false gauge showing

### Actual Result

shows i always have 990 gallons of propane. i no longer have tank monitor. if i switch fuel to nat gas on the genmon program, the gauge goes away. my question is....does this switch mess with any other values generated. i know nat gas units are rated lower than those run on propane. by switching the system to say its running on nat gas effect any of the readings i see on my monitor screen?

### Version Info

V1.19.02

**Comment by jgyates** [comment, marked as answer]
I converted this "issue" to a "discussion". Github now has discussion areas and issues are for but reports or feature requests only.

First, I would need clarification on which fuel gauge you are talking about? There is an estimated fuel gauge, there are fuel gauges for actual sensors (tank utility, DIY gauge, Liquid cooled diesels have a fuel gauge built in).

There are a few different ways a fuel gauge can show up, depending on your controller and what add ons you have enabled. If you are using a propane tank monitor, then simply disable the add on for the tank monitor on the add on page. If you don't wan't and estimated tank level, then set your tank size to zero in the settings page

If that does not solve the issue for you then I will need to see your log.

**Comment by grsthegreat** [comment]
Thank you, that answered my questions. I had the tank alert system that failed. The new tankalert monitors REQUIRE yearly fees to monitor and tied to generacs own monitoring system. Personally, i would not get another one as the one i had failed after 3 years when battery dies, and new batteries would not reset it. Their tech support said the units generally fail before batteries die. Some product.

glad i never recommended or sold these when i was installing generators.

**Comment by jgyates** [comment]
I have a tank utility monitor on a propane tank not connected to a generator. It had to be replaced once and always reports low battery even with brand new batteries.  Also, updates to the tank reading over are relatively quick if you use their app, but if you try to read the data over their (documented) API the updates take about 36-48 hours to be able to read the new reading. Their product is a bit lacking in my view. From an outsiders point of view I think they are a dying company that is looking for a new revenue stream.
